src/util/Permissions.js

```diff
@@ -310,6 +215,11 @@ Permissions.prototype.hasPermissions = util.deprecate(Permissions.prototype.hasP
   'EvaluatedPermissions#hasPermissions is deprecated, use Permissions#has instead');
 Permissions.prototype.missingPermissions = util.deprecate(Permissions.prototype.missingPermissions,
   'EvaluatedPermissions#missingPermissions is deprecated, use Permissions#missing instead');
+Object.defineProperty(Permissions.prototype, 'raw', {
+  get: util
+    .deprecate(Object.getOwnPropertyDescriptor(Permissions.prototype, 'member').get,
```

Comment:
```suggestion
    .deprecate(Object.getOwnPropertyDescriptor(Permissions.prototype, 'bitfield').get,
```

As per deprecation description.

Reply:
This isn't correct, as Permissions#bitfield isn't a getter

Reply:
That should have been 'raw', as that's the deprecated getter.
Fixed that and also deprecated the setters now :eyes: